### ISSUE TITLE
Get flooded with errors, but seemingly no problem (Godot 3.3)

### ISSUE THREAD

**Issue by SaffronStreams** [godotengine/godot#48294]
I am using Windows 10 and I am trying out Godot 3.3 stable on my project and getting the following error:

E 0:00:05.217   is_action_pressed: Request for nonexistent InputMap action 'play_right'.
  <C++ Error>   Condition "!has_action" is true. Returned: false
  <C++ Source>  main/input_default.cpp:102 @ is_action_pressed()
  <Stack Trace> MovementState.gd:103 @ process_input()
                MovementState.gd:68 @ update()
                Player.gd:105 @ _process()

I get this for seemingly all of my custom-defined inputs (flooding, as they get output on each frame), but at the same time have no trouble using these inputs to play the game. However, these are red errors and I remember it's best to fix these asap, and so the question becomes are these actual errors or some 3.3 problem.

The original InputMap action were defined in a 3.2.3 stable project and the project was then opened without changing anything in 3.3.

**Comment by akien-mga** [comment]
You probably have a `tool` script that tries to process input while in the editor, which is invalid. Your game's input mappings are available at runtime, but not in the editor itself.

Before 3.3 it would fail silently, now it reports failures.

**Comment by SaffronStreams** [comment]
I am running "vanilla" Godot 3.3, I just defined the inputs in the Input Map tab in Project settings and use "if Input.is_action_pressed("play_left")" to get Inputs, so I am not consciously running a tool script, unless I misunderstand what that is. Is there any way to list additional scripts that are running?

**Comment by Xrayez** [comment]
The error:
> E 0:00:05.217 is_action_pressed: Request for nonexistent InputMap action 'play_right'.

tells that you don't actually have the 'play_right' defined in the project settings. If this is the case, it's surprising that you're able to get inputs with `Input.is_action_pressed("play_left")` in the first place.

Make sure you're not mixing words: "play_right", "play_left", "left_play", "right_play" etc.

It's difficult to tell what goes wrong without a project reproducing the issue reported.

**Comment by SaffronStreams** [comment]
TLDR: problem solved, issue resolved?, I have made a mistake in verifying earlier.

After trying to duplicate unsuccessfully, I re-examined if I had defined them or not. I can go left and right and aim so "they are obviously defined", but looking closer I found not all of the controls gave errors and then I realized, like a dummy that it's only the controls for which I'm currently using control sticks on the controller, and the above definitions were for backups on keyboard I will put in later.

Humble apologies, friends.
